lib/advisor/core/advice/advice_finder.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Advisor.Core.AdviceFinder do
```

Comment:
The name here doesn't match the file hierarchy exactly. To be more accurate I would have
```
Advisor.Core.Advice.AdviceFinder
```

at which point you may want to rename it to:
```
Advisor.Core.Advice.Finder
```